drivers/can/can_shell.c

```diff
@@ -519,6 +519,102 @@ static int cmd_can_dbitrate_set(const struct shell *sh, size_t argc, char **argv
 	return 0;
 }
 
+static int can_shell_parse_timing(const struct shell *sh, size_t argc, char **argv,
+				  struct can_timing *timing)
+{
+	char *endptr;
+
+	timing->sjw = (uint32_t)strtoul(argv[2], &endptr, 10);
```

Comment:
It may be worth considering using shell_strtoul here, e.g:
https://github.com/zephyrproject-rtos/zephyr/blob/7a3a6d0c030b3bc99e669797ceb387d404331ce0/subsys/bluetooth/mesh/shell/sar.c#L42-L48

Reply:
Thank you for the suggestion. The rest of this file already make use of `strtoul()` and the additional error checking is not really needed here. If `*endptr != '\0'` after the call to `strtoul()` the string is valid and was converted. This is all the error checking needed here.